packages/two_dimensional_scrollables/example/lib/tree_view/custom_tree.dart

```diff
@@ -0,0 +1,275 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:two_dimensional_scrollables/two_dimensional_scrollables.dart';
+
+/// The class containing a TreeView that highlights the selected node.
+/// The custom TreeView.treeNodeBuilder makes tapping the whole row of a parent
+/// toggle the node open and closed with TreeView.toggleNodeWith. The
+/// scrollbars will appear as the content exceeds the bounds of the viewport.
+class CustomTreeExample extends StatefulWidget {
```

Comment:
Looks like the keyboard navigation isn't supported. Is this planned for later?

Reply:
In this sample, the children rendered in the tree rows don't have focus nodes, so no, the keyboard won't traverse them If you build buttons in the tree rows though, as an example, keyboard navigation works as expected.